jobs/payment-jobs/tasks/unpaid_statement_notify_task.py

```diff
@@ -0,0 +1,131 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task to notify user for any outstanding statement."""
+from datetime import timedelta
+
+from flask import current_app
+from pay_api.models import db
+from pay_api.models.invoice import Invoice as InvoiceModel
+from pay_api.models.payment_account import PaymentAccount as PaymentAccountModel
+from pay_api.models.statement import Statement as StatementModel
+from pay_api.models.statement_invoices import StatementInvoices as StatementInvoicesModel
+from pay_api.models.statement_recipients import StatementRecipients as StatementRecipientsModel
+from pay_api.models.statement_settings import StatementSettings as StatementSettingsModel
+from pay_api.services.flags import flags
+from pay_api.utils.enums import InvoiceStatus, PaymentMethod, StatementFrequency
+from pay_api.utils.util import current_local_time, get_first_and_last_dates_of_month
+from sentry_sdk import capture_message
+from sqlalchemy import func
+
+from utils.mailer import publish_payment_notification
+
+
+class UnpaidStatementNotifyTask:
```

Comment:
Can we add in a blurb, noting this is for EFT only?

Reply:
Added.